tests/MongoDB.Analyzer.Tests.Common.TestCases/Builders/BuildersBasic.cs

```diff
@@ -320,5 +320,125 @@ public void Multiple_expression_variables_declaration_and_reassignment()
             x = y = Builders<User>.Filter.Lt(u => u.Age, 15) & Builders<User>.Filter.Gt(u => u.Age, 65);
             x = z = w = y = x = z = w = y = Builders<User>.Filter.Lt(u => u.Age, 17) & Builders<User>.Filter.Gt(u => u.Age, 18);
         }
+
+        [BuildersMQL("{ \"Age\" : 1 }")]
+        public void builders_projection_1()
```

Comment:
1. No need for "builders" prefix. Also start the methods names with a capital letter.
2. Also you can try grouping similar tests to a single test method, for examples seems that Includes can reside together.
3. Lets move those tests to `BuildersProjection.cs` file.

Reply:
Resolved